src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -2279,12 +2279,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         hasErrors = True
                     Else
                         Dim useSiteDiagnostics As HashSet(Of DiagnosticInfo) = Nothing
+                        Dim accessThroughType = GetAccessThroughType(actualEventAccess.ReceiverOpt)
+                        If Not (Me.IsAccessible(method, useSiteDiagnostics, accessThroughType) OrElse
+                            Not IsAccessible(eventSymbol, useSiteDiagnostics, accessThroughType)) Then
```

Comment:
Agreed, we should only need the first check: `IsAccessible(method, useSiteDiagnostics, accessThroughType)`.